### PR TITLE
awsemf-exporter: replace pattern "{{ }}" with "{ }"

### PR DESCRIPTION
**Pull request by hossain-rayhan**
This contribution replace `ClusterName` and `TaskId` placeholder pattern `{{ }}` with `{ }`. AWS SSM parameter doesn't support the pattern `{{ }}`- so we need to update this.

Related Issue:
#1662 

Test:
Unit tests updated.

### CODE REVIEW

**Comment by hossain-rayhan** [comment]
@bogdandrutu and @tigrannajaryan can anybody please help me to get this reviewed and merged today. Its a one line change- replace `{{ }}` with `{ }`. Will highly appreciate.

File: exporter/awsemfexporter/emf_exporter_test.go [hunk 1]
```diff
@@ -335,7 +335,7 @@ func TestConsumeMetricsWithWrongPlaceholder(t *testing.T) {
 	expCfg.Region = "us-west-2"
 	expCfg.MaxRetries = defaultRetryCount
 	expCfg.LogGroupName = "test-logGroupName"
-	expCfg.LogStreamName = "{{WorngKey}}"
+	expCfg.LogStreamName = "{WorngKey}"
```
Comment (kbrockhoff) [inline exporter/awsemfexporter/emf_exporter_test.go:338]: Need to fix spelling to match below

File: exporter/awsemfexporter/emf_exporter_test.go [hunk 2]
```diff
@@ -391,7 +391,7 @@ func TestConsumeMetricsWithWrongPlaceholder(t *testing.T) {
 	require.NoError(t, exp.Shutdown(ctx))
 	streamToPusherMap, ok := exp.(*emfExporter).groupStreamToPusherMap["test-logGroupName"]
 	assert.True(t, ok)
-	pusher, ok := streamToPusherMap["{{WorngKey}}"]
+	pusher, ok := streamToPusherMap["{WorngKey}"]
```
Comment (kbrockhoff) [inline exporter/awsemfexporter/emf_exporter_test.go:394]: Need to fix spelling to match below

**Review by kbrockhoff — CHANGES_REQUESTED** [review]

File: exporter/awsemfexporter/emf_exporter_test.go (hunk 1 above)
Comment (hossain-rayhan) [reply exporter/awsemfexporter/emf_exporter_test.go]: Updated.

File: exporter/awsemfexporter/emf_exporter_test.go (hunk 2 above)
Comment (hossain-rayhan) [reply exporter/awsemfexporter/emf_exporter_test.go]: Updated.

**Review by kbrockhoff — APPROVED** [review]

**Comment by kbrockhoff** [comment]
@jrcamp this is ready to merge

**Review by anuraaga — APPROVED** [review]

**Pull request merged**